src/util/git.js

```diff
@@ -210,7 +210,7 @@ export default class Git implements GitRefResolvingInterface {
       if (await Git.repoExists(secureRef)) {
         return secureRef;
       } else {
-        reporter.warn(reporter.lang('downloadHTTPWithoutCommit', ref.repository));
+        reporter.warn(reporter.lang('getDownloadHttpWithoutCommit', ref.repository));
```

Comment:
Shouldn't this be `gitDownloadHttpWithoutCommit` (rather than `get`)?

Reply:
Yes, I will update.